AmountInput/README.md

```diff
@@ -0,0 +1,47 @@
+
+# AmountInput Interface
+
+**Status**: Proposed
+
+| Name        | ID                                                                |
+| :---------- | :---------------------------------------------------------------- |
+| AmountInput | a1d347099e29c1624c8890619daf207bde18e92df5220a54bcc6d858309ece84  |
+
+
+## Description
+
+Interface for amount input.
+
+## Functions
+
+`get` - prints prompt message to the user and returns number of nanotons entered by user.
+
+arguments:
+
+	answerId: uint32
+    prompt: bytes
+    step: uint256
+	min: uint256
+	max: uint256
```

Comment:
```suggestion
    answerId: uint32 - id of callback function.
    prompt: bytes - utf-8 string to print to the user before input.
    decimals: uint8 - number decimals that input currency has, e.g. 9 - means browser should multiply the amount to 1000000000 to get its debot representation.
    min: uint256 - minimal allowed amount.
    max: uint256 - maximum allowed amount.
```

Reply:
[amount_input.tar.gz](https://github.com/tonlabs/DeBot-IS-consortium/files/5980657/amount_input.tar.gz)
AmountInput.sol
AmountInput.abi.json
AmountInput.hpp
Example.sol